ApplicationRequirements.txt

```diff
@@ -0,0 +1,139 @@
+"Log-in" Screen
+	"Email" Text Box
+	"Password" Text Box
+	"Sign-in" Button
+		-> Takes to "Home" Screen if correct
+			-> Takes to the Driver variant of the "Home" Screen if account is registered to a driver
+		-> Redirects back to "Log-in" Screen if incorrect
+			-> Notifies user with red text
+	"Create New Account" Button
+		-> Takes to "Create New Account" Screen
+		
+"Home" Screen - General
+	"Create New Request" Button
+		-> Takes to "Create New Request" Screen
+	"My Requests" Button
+		-> Takes to "My Requests" Screen
+		
+"Home" Screen - Driver
+	"Create New Request" Button
+		-> Takes to "Create New Request" Screen
+	"My Requests" Button
+		-> Takes to "My Requests" Screen
+	"Requests Available for Pickup" Button
+		-> Takes to "Requests Available for Pickup" Screen
+	"My Deliveries" Button
+		-> Takes to "My Deliveries" Screen
+		
+"Create New Account" Screen
+	"Name" Text Box
+	"Email" Text Box
+	"Phone Number" Text Box
+	"Home Address" Text Box
+	"New Password" Text Box
+	"Confirm Password" Text Box
+	"Create New Account" Button
+		-> Takes to "Home" Screen if information above is valid
+			-> All boxes completed
+			-> Email not already attached to account
+			-> "New Password" and "Confirm Password" text boxes contain same password
+		Redirects back to "Create New Account" Screen if above criteria is not met
+			-> Notifies user what went wrong in red text
+
+"My Requests" Screen
+	Shows the "Request History" of the user
+	Each request shows the status of each request (en route, delivered, awaiting driver, canceled, etc.)
+	Each request shows all of the information inputted when creating the request (deliver to, pickup from, etc.)
+	Each request has an "Update Request" Button
+		-> Takes to "Update Request" Screen if not already out for delivery
+		-> Requests can NOT be updated if out for delivery
+	Each request has a "Cancel" Button
+		-> Displays "Confirm Cancel" Pop-up
+		-> Displays "Confirm Cancel" Button
+			-> Changes status of request to "Canceled" if not already out for delivery
+			-> Request is removed from the "Request Queue" if not already out for delivery
+		-> Requests can NOT be canceled if out for delivery
+			
+"Create New Request" Screen
+	"Deliver To" Text Box
+	"Delivery Address" Text Box
+	"Delivery Instructions" Text Box (Optional: For preferred delivery time and other special instructions)
+	"Pickup From" Text Box
+	"Pickup Address" Text Box
+	"Pickup Instructions" Text Box (Optional: For preferred pickup time and other special instructions)
+	"What is Being Delivered" Text Box
+	"Upload Files" Button (Optional: For uploading receipt/confirmation for online order)
+		-> Opens "File Explorer" to choose file
+	"Checkout" Button
+		-> Takes user to "Checkout" Screen
+	"Cancel" Button
+		-> Displays "Confirm Cancel" Pop-up
+		-> Displays "Confirm Cancel" Button
+			-> Takes user to "Home" Screen
+		
```

Comment:
Are we not going to add delivery time and pickup time. For users that are looking for a timed delivery.

Reply:
I had it on there but figured that’d just be something that can be put in the delivery instructions. It can be added back if we want that.